_component_design/choice-tiles.md

```diff
@@ -239,22 +264,22 @@ For selected states, the iconography changes color to match the given state. In
 
 <section class="static-section" markdown="1">
 
-## Handling errors
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Single-select error
+### Disabled
+
+When possible, inform the user about why an option is not available. Consider adding a tooltip to a disabled choice tile explaining why it is disabled and how to enable it.
 
-If there is an error on a selection, change that item to errored state and display an error message below it.
+When you cannot control the color of the icon, set the icon opacity to 30%.
```

Comment:
did this get approved by Ryan? we should check on this.

Reply:
on 1/29 we will discuss in bowling.